crates/bevy_ecs/src/world/mod.rs

```diff
@@ -447,6 +447,8 @@ impl World {
     ///
     /// To iterate over entities in a deterministic order,
     /// sort the results of the query using the desired component as a key.
+    /// Note that it requires fetching the whole result set from the query
```

Comment:
```suggestion
    /// Note that this requires fetching the whole result set from the query
```
"this" is clearer and more natural than "it" here in English <3